### PR TITLE
Add touch drift/browser margins test page

### PR DESCRIPTION
**Pull request by llewelld**
Adds a manual test page that's intended to help testing various aspects
of page display with page margins and the toolbar:

1. Are fixed elements correctly moved if the toolbar/keyboard is open.
2. Does element touch selection drift when the page is scrolled.
3. Are all elements (fixed and scrollable) fully visible on the page.

### CODE REVIEW

File: tests/manual/test-margin-drift.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+				<br/><a href="#">Seventh link</a>
+				<br/><a href="#">Eighth link</a>
+			</div>
+			<br/>
+			<br/>
+			<br/>
+			<br/>
+			<br/>
+			<div style="position:absolute; border: red solid; left: 33%; width: calc(66% - 10px);">
+			<p/>Test page margins. Left fixed; right scrollable.
```
Comment (okodron) [inline tests/manual/test-margin-drift.html:38]: I guess indention is not so accurate here inside `<div>`

Comment (llewelld) [reply tests/manual/test-margin-drift.html]: Thanks, and nicely spotted. It should now be fixed.

**Review by okodron — APPROVED** [review]

File: tests/manual/testmargindrift.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+		<meta name="viewport" content="initial-scale=1">
+	</head>
+		<body>
```
Comment (rainemak) [inline tests/manual/testmargindrift.html:7]: Please do not use tabs to indent, normal 4 spaces for each indentation level.

Comment (llewelld) [reply tests/manual/testmargindrift.html]: Thanks; this should be fixed now.

**Review by rainemak — APPROVED** [review]
:+1:

**Comment by llewelld** [comment]
Rebased and fixed conflict.

**Pull request merged**